server/modules/analysis/repository/processingStepCalculationRepository.js

```diff
@@ -2,24 +2,25 @@ import * as R from 'ramda'
 
 import camelize from 'camelize'
 
-import * as ProcessingStep from '@common/analysis/processingStep'
 import * as ProcessingStepCalculation from '@common/analysis/processingStepCalculation'
 
 import { db } from '@server/db/db'
 import * as DbUtils from '@server/db/dbUtils'
 
 import { getSurveyDBSchema } from '@server/modules/survey/repository/surveySchemaRepositoryUtils'
 
+const _selectFields = `uuid, processing_step_uuid, index, node_def_uuid, props`
+
```

Comment:
have you checked fetchStepsAndCalculations in stepRepository?

Reply:
fixed